action.yml

```diff
@@ -175,6 +197,16 @@ runs:
         fi
       shell: bash
 
+    - name: Run Tutor extra commands
+      if: ${{ inputs.tutor_extra_commands_path }}
+      run: |
+        source .tutor_venv/bin/activate
+
+        cd ${{ inputs.app_name }}
+        chmod +x ./${{ inputs.tutor_extra_commands_path }}
+        ./${{ inputs.tutor_extra_commands_path }}
+      shell: bash
```

Comment:
Should we document this step's usage in the readme file?

Reply:
Yes, I added the new step in the readme: https://github.com/eduNEXT/integration-test-in-tutor/commit/ece36e62bcc8bcfda5de87af78a699029279fb9f